vtk_viewer/test/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+find_package(GTest REQUIRED)
+
+add_executable(${PROJECT_NAME}_unit utest.cpp)
+target_link_libraries(${PROJECT_NAME}_unit PRIVATE GTest::GTest GTest::Main ${PROJECT_NAME})
+target_clang_tidy(${PROJECT_NAME}_unit)
+add_code_coverage(${PROJECT_NAME}_unit ALL EXCLUDE ${COVERAGE_EXCLUDE})
+add_gtest_discover_tests(${PROJECT_NAME}_unit)
+add_dependencies(${PROJECT_NAME}_unit ${PROJECT_NAME}_unit)
```

Comment:
Should this be: `add_dependencies(${PROJECT_NAME}_unit ${PROJECT_NAME})`

Reply:
fixed